devicetypes/smartthings/zwave-water-sensor.src/zwave-water-sensor.groovy

```diff
@@ -84,6 +84,9 @@ def configure() {
 		// Tell sensor to send us battery information instead of USB power information
 		commands << encap(zwave.configurationV1.configurationSet(parameterNumber: 0x5E, scaledConfigurationValue: 1, size: 1))
 		response(delayBetween(commands, 1000) + ["delay 20000", encap(zwave.wakeUpV1.wakeUpNoMoreInformation())])
+	} else if (zwaveInfo.mfr.equals("021F") || zwaveInfo.mfr.equals("0258")) {
+		// wakeUpInterval set to 4 h for NEO Coolcam and Dome
+		zwave.wakeUpV1.wakeUpIntervalSet(seconds: 4 * 3600, nodeid: zwaveHubNodeId).format()
```

Comment:
This hub should actually be automatically setting the wakeupinterval to 4 hours already so this code should be unnecessary. 

But since you're changing the checkInterval values I want you to verify that these devices do actually honor the 4 hour wakeupinterval.

Reply:
Why the hub should be automatically setting 4 hours wakeupinterval? I thought that I have to set this manually because devices had before 12 hours wakeupinterval by default. 
And yes, I verified this in the very begging to be sure it is working.

Reply:
It's just part of our add process for wakeup devices. The default is 4 hours. This shouldn't be necessary.

Reply:
Good, should be fine now.